### PR TITLE
React capstone project - Metrics webapp: Pull Request

### PR DESCRIPTION
**Pull request by belaymit**
# Project Features

## Project requirements
- The development of this web app is based on Crypto currencies  API

## Design

- Colors - ```light mode``` and ``dark mode``.
- Typography: ``Poppins``, ``Nunito`` and  ``Roboto``
## Interactions

- Home page
- [ ] When the page loads, the web app shows the list of cryptocurrencies/categories that could be filtered out by name, symbol, or rank.
- [ ] Along with the category name, Symbol, Price, and daily changes per cryptocurrency that come from the API are displayed
- [ ] When the user clicks (or taps) on a category item, the application navigates to the ``details page``.

- Details page
- [ ] In the details page, the web app retrieves data from the API to show detailed data in the category.
- [ ] When the user clicks on the "Back" button (<), the user navigates to the home page.

## Testing requirements

- [ ] Created unit tests for pure functions in your Redux code
- [ ] Created integration tests for your application using the React Testing Library

## Technical requirements

- [ ] The project is a single-page application (SPA) built with React and Redux.
- [ ] The data retrieved from the API should be stored in the Redux store.
- [ ]  Filtered the data that is retrieved from the API using a Filter stateless component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,51 +186,55 @@ Example:
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+I would like to thank Microverse community
```
Comment (fobadara) [inline README.md]: [REQUIRED] Kindly credit Nelson Sakwa the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/pages/home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                <h2 className="crypto-container-title">Top 100 Cryptocurrencies</h2>
+                <div className="card-container">
+                  {filteredCoins.map((coin) => (
+                    <Card
+                      key={coin.id}
+                      coin={coin}
+                    />
+                  ))}
+                </div>
+              </>
```
Comment (fobadara) [inline src/pages/home/Home.jsx:97]: [REQUIRED] Kindly follow the template. There should not be any space between or beside your cards

image

File: README.md [hunk 3]
```diff
@@ -237,51 +186,55 @@ Example:
...
-  - [Answer_2]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```
Comment (fobadara) [inline README.md:237]: [OPTIONAL] Kindly add your details like your name to your MIT license

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @belaymit, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good first attempt
✅ Project looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @belaymit,

✔️ Implementation of design

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**